.github/00-bug-performance-issue.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug/Performance Issue
+about: Use this template for reporting a bug or a performance issue.
+
+---
+
+tag:bug_template</em>
+
+**Please provide the following information**
+
+**System information**
+- Tensorboard version(from pip package, also printed out when running `tensorboard`) :
```

Comment:
nit: can we avoid spaces before the trailing colons, on all lines in all templates?